blog-service/2024-12-04-search.md

```diff
@@ -0,0 +1,16 @@
+---
+title: compareCIDRPrefix and getCIDRPrefix Operators Behavior Change (Search)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - compareCIDRPrefix
+  - getCIDRPrefix
+  - search-operator
+  - log-search
+hide_table_of_contents: true  
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>
+
+We are happy to announce the behavioural change for the `compareCIDRPrefix` and `getCIDRPrefix` operators. Previously, these operators would fail if there were trailing or leading spaces around the IP address. With this update, the `compareCIDRPrefix` and `getCIDRPrefix` operators have become more flexible and can do trimming, which means the operators will not fail even if there are trailing or leading spaces around the IP address.
```

Comment:
```suggestion
We are happy to announce the behavioural change for the [`compareCIDRPrefix`](/docs/search/search-query-language/search-operators/cidr/#comparecidrprefix) and [`getCIDRPrefix`](/docs/search/search-query-language/search-operators/cidr/#getcidrprefix) operators. Previously, these operators would fail if there were trailing or leading spaces around the IP address. With this update, the `compareCIDRPrefix` and `getCIDRPrefix` operators have become more flexible and can do trimming, which means the operators will not fail even if there are trailing or leading spaces around the IP address.
```